Dockerfile

```diff
@@ -1,10 +1,12 @@
 FROM python:3 as app
 ENV PYTHONUNBUFFERED 1
 RUN pip install --upgrade pipenv gunicorn
+RUN pip install pymysql
```

Comment:
The PostgreSQL library is installed as part of the base Pipenv requirements and I _think_ we can do the same thing with the `pymysql` library because, unlike `mysqlclient`, it doesn't need to compile against mysql client libraries.

So lets get this instead in to Pipenv, which you can do by running `pipenv install pymysql` from the repo root and committed the changes to `Pipfile` and `Pipfile.lock`.

Reply:
So when I try to run this, I get the following output:
```
Creating a virtualenv for this project…
Using /usr/bin/python3 (3.8.2) to create virtualenv…
⠋/usr/bin/python3: No module named pipenv.pew

Virtualenv location: 
Installing pymysql…
⠼
Error:  An error occurred while installing pymysql!
/bin/sh: 1: /bin/pip: not found
```

Reply:
I went back to the "Installation" portion of the contributing doc, and found the steps to setup the environment.

I'm trying to run 
```
# pipenv install --three
Creating a virtualenv for this project…
Using /usr/bin/python3 (3.8.2) to create virtualenv…
⠋/usr/bin/python3: No module named pipenv.pew

Virtualenv location: 
Creating a virtualenv for this project…
Using /usr/bin/python3 (3.8.2) to create virtualenv…
⠙/usr/bin/python3: No module named pipenv.pew

Virtualenv location: 
Installing dependencies from Pipfile.lock (f8bb4c)…
Traceback (most recent call last):▉▉▉▉▉ 0/42 — 00:00:00
  File "/usr/bin/pipenv", line 11, in <module>
    load_entry_point('pipenv==11.9.0', 'console_scripts', 'pipenv')()
  File "/usr/lib/python3/dist-packages/pipenv/vendor/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib/python3/dist-packages/pipenv/vendor/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/usr/lib/python3/dist-packages/pipenv/vendor/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/lib/python3/dist-packages/pipenv/vendor/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/lib/python3/dist-packages/pipenv/vendor/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/usr/lib/python3/dist-packages/pipenv/cli.py", line 349, in install
    core.do_install(
  File "/usr/lib/python3/dist-packages/pipenv/core.py", line 1875, in do_install
    do_init(
  File "/usr/lib/python3/dist-packages/pipenv/core.py", line 1362, in do_init
    do_install_dependencies(
  File "/usr/lib/python3/dist-packages/pipenv/core.py", line 832, in do_install_dependencies
    c = pip_install(
  File "/usr/lib/python3/dist-packages/pipenv/core.py", line 1477, in pip_install
    c = delegator.run(pip_command, block=block)
  File "/usr/lib/python3/dist-packages/pipenv/vendor/delegator.py", line 267, in run
    c.run(block=block, binary=binary)
  File "/usr/lib/python3/dist-packages/pipenv/vendor/delegator.py", line 156, in run
    s = PopenSpawn(self._popen_args, **pexpect_kwargs)
  File "/usr/lib/python3/dist-packages/pipenv/vendor/pexpect/popen_spawn.py", line 46, in __init__
    self.proc = subprocess.Popen(cmd, **kwargs)
  File "/usr/lib/python3.8/subprocess.py", line 854, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/usr/lib/python3.8/subprocess.py", line 1702, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: '/bin/pip'
     ▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉ 0/42 — 00:0
```

Reply:
Looks like pip is missing? What do you get for the following commands? --

```
python -v
pip -v
python3 -v
pip3 -v
pipenv -v
```

Reply:
**python -V**
Python 2.7.18rc1
**pip -v** 
not found
**python3 -V**
Python 3.8.2
**pip3 -V**
pip 20.0.2 from /usr/lib/python3/dist-packages/pip (python 3.8)
**pipenv --version**
pipenv, version 11.9.0

Reply:
Hm, I have seen people in the past have to alias `python` to `python3` to make Pipenv happy. You could try ` alias python=python3` before running the commands (and maybe also `alias pip=pip3`).

Alternatively maybe take a look at this reference for the `No module named pipenv.pew` error -- which I have never seen before: pypa/pipenv/issues/2557